overlord/ifacestate/handlers.go

```diff
@@ -35,6 +35,15 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+// optsFromFlags returns interfaces.ConfinementOptions from snapstate.Flags.
+func optsFromFlags(flags snapstate.Flags) interfaces.ConfinementOptions {
```

Comment:
confinementOptions or similar would be a better name here. optsFromFlags says nothing about what it's taking or what it's returning, so when we read something like this:
```
opts := optsFromFlags(flags)
```
It's completely impossible to tell what this is about without looking elsewhere.